infrateam_first_responder.md

```diff
@@ -54,7 +54,7 @@ Use the `sdr-deploy` script to deploy all infrastructure projects (with **import
 
 Note that you will need to be sure you can ssh into each of the VMs from wherever you are running the deploy script.
 
-- qa: deploy to qa with script
+- qa: deploy to qa first with script, then run infrastructure-integration-tests after deploy to qa.  Note as below that some tests may fail but will pass if run individually.  If on a Mac, you will get better results if you stay in the same "space" as the running tests (avoids focus issues with the browser).
```

Comment:
Why run the integration tests two times in different environments? What is the objective here?  This seems like it will just be extra work for the FR, preventing them from working on other things and not deliver any additional value.

Reply:
so which one does it make more sense to run on...qa?  i don't have a preference, just want this to be clear from our future selves

Reply:
I think that running everything on stage has been the way we have been doing it (because QA was not complete).   I don't see any reason we need to change from that practice.

Reply:
ok, also just noted that the H2 spec looks for public XML, which currently fails in QA since there is no purl-qa